audio_service/android/build.gradle

```diff
@@ -22,7 +22,7 @@ rootProject.allprojects {
 apply plugin: 'com.android.library'
 
 android {
-    compileSdkVersion 28
```

Comment:
Don't remember why exactly I bumped it up, some constants weren't show up in IDE I guess. But why not